tests/PHPStan/Analyser/data/array-key-exists.php

```diff
@@ -12,20 +12,89 @@ class Foo
 	 * @param array<string, string> $a
 	 * @return void
 	 */
-	public function doFoo(array $a, string $key): void
+	public function doFoo(array $a, string $key, int $anotherKey): void
 	{
 		assertType('false', array_key_exists(2, $a));
 		assertType('bool', array_key_exists('foo', $a));
 		assertType('false', array_key_exists('2', $a));
 
-		$a = ['foo' => 2, 3 => 'bar'];
```

Comment:
Please do not change existing tests like this. This diff is hard to review.

Of course, if you improved behaviour and existing asserts changed, you need to edit them, but otherwise add new tests instead. I don't want to change meaning of variables here, please continue testing `$a` there as it was tested before.

Reply:
Sure, I changed to keep the old name of the variable.
I also rebase the branch on 1.9.x.